docs(companion): add 2 new env descriptions (#345)
transloadit/uppy.io: docs/companion.md

will prefix all events with the name provided and a colon. See also
`COMPANION_REDIS_EXPRESS_SESSION_PREFIX`.

#### `maxFilenameLength` `COMPANION_MAX_FILENAME_LENGTH`

> Maximum allowed length for uploaded filenames. Default: 500. If the max length is set to `0`, the filename will be fully truncated and become an UUID. For example, if the filename is `my-long-filename.txt` and the max length is set to `0`, the filename will become `a7c73ddb-3dd7-404d-937a-8b836ff099d8`.

Reviewer: ```suggestion
Maximum allowed length for uploaded filenames. Default: 500. If set, it must be set to a value larger than `0`.
```
